src/statmodels.jl

```diff
@@ -145,11 +145,18 @@ end
 
 const adjr² = adjr2
 
+
```

Comment:
Unrelated change here. Would be great if you could remove it, though it's not a huge deal.